### PR TITLE
Update Kind, Istio, Helm, and Kubectl versions

### PR DESCRIPTION
**Pull request by rjeberhard**

### CODE REVIEW

**Review by marinakog — CHANGES_REQUESTED** [review]
need to change HELM_VERSION in Jenkinsfile.oke, also current default version for Istio is 1.16.1 (search for ISTIO_VERSION) for oke runs, should we also update it to 1.17?

**Comment by rjeberhard** [comment]
> need to change HELM_VERSION in Jenkinsfile.oke, also current default version for Istio is 1.16.1 (search for ISTIO_VERSION) for oke runs, should we also update it to 1.17?

@marinakog, done. What's the current way to kick-off an OKE run to validate?

File: Jenkinsfile [hunk 1]
```diff
@@ -225,15 +245,17 @@ pipeline {
         )
         string(name: 'KUBECTL_VERSION',
                description: 'kubectl version',
-               defaultValue: '1.21.5'
```
Comment (anpanigr) [inline Jenkinsfile:228]: Why we are downgrading the KUBECTL_VERSION from 1.21.5 to 1.25.4 ?

Comment (rjeberhard) [reply Jenkinsfile:228]: This is an upgrade from 1.21.5 to 1.25.4.

**Comment by marinakog** [comment]
> > need to change HELM_VERSION in Jenkinsfile.oke, also current default version for Istio is 1.16.1 (search for ISTIO_VERSION) for oke runs, should we also update it to 1.17?
> 
> @marinakog, done. What's the current way to kick-off an OKE run to validate?

I have started run https://build.weblogick8s.org:8443/job/wko-oke-nightly-seqential/375/ ( you need to change branch via configure if you want to try different from main branch)

**Review by anpanigr — COMMENTED** [review]
Do we need to change the Doc file ?

**Comment by marinakog** [comment]
> > > need to change HELM_VERSION in Jenkinsfile.oke, also current default version for Istio is 1.16.1 (search for ISTIO_VERSION) for oke runs, should we also update it to 1.17?
> > 
> > 
> > @marinakog, done. What's the current way to kick-off an OKE run to validate?
> 
> I have started run https://build.weblogick8s.org:8443/job/wko-oke-nightly-seqential/375/ ( you need to change branch via configure if you want to try different from main branch)

it failed due   apiVersion: v1
    kind: Pod
    fieldPath: spec.containers{fix-pvc-owner}
    name: change-permissions-onpv-job-mii-jms-istio-db-pv-wgbtub-h7b8g
    namespace: ns-vytpry
    resourceVersion: '7173'
    uid: 4cadaa49-428d-498c-911b-49517ab0b998
  lastTimestamp: '2023-04-06T17:46:48Z'
  message: Back-off pulling image "phx.ocir.io/weblogick8s/test-images/weblogic:12.2.1.4"
  reason: BackOff
  reportingComponent: ''
  reportingInstance: '',    I have restarted run see https://build.weblogick8s.org:8443/job/wko-oke-nightly-seqential/376/

**Review by marinakog — APPROVED** [review]

**Comment by marinakog** [comment]
latests results on that branch https://build.weblogick8s.org:8443/job/wko-oke-nightly-seqential/377/

**Pull request merged**